ansible/roles/apigateway/tasks/deploy.yml

```diff
@@ -28,3 +29,6 @@
   until: result.status == 200
   retries: 12
   delay: 5
+
+- name: install tzdata on apigateway
+  shell: docker exec apigateway sh -c "apk add -u tzdata && api-gateway -s reload"
```

Comment:
Should this be pushed upstream rather than patching here?

Reply:
yes, that's better, but then it won't compatible with old apigateway